aks/application/variables.tf

```
@@ -102,3 +102,20 @@ variable "github_personal_access_token" {
   default     = null
   description = "Github Personal Access Token (PAT) of github_username"
 }
+
+variable "azure_resource_prefix" {
+  type        = string
+  default     = null
+  description = "Prefix of Azure resources for the service"
+}
+
+variable "service_short" {
+  type        = string
+  default     = null
+  description = "Short name of the service"
+}
+
+variable "azure_enable_container_monitoring" {
```

```suggestion
variable "azure_enable_monitoring" {
```

Can I suggest this name to match the naming convention we've used for the Redis and Postgres modules? I realise it's less specific but I'm not sure `container` adds anything here as we're already within the module for an application deployment.

---

changed as requested